src/languages.js

```diff
@@ -7,6 +7,7 @@ const languages = [
   { short: 'fa', long: 'Farsi', rtl: true },
   { short: 'fr', long: 'Français', rtl: false },
   { short: 'tr', long: 'Turkish', rtl: false },
+  { short: 'sl', long: 'Slovenščina', rtl: false },
```

Comment:
This list is sorted alphabetically, except for English (which is first). So please swap Turkish and Slovenščina :)

Reply:
@LukasKalbertodt .. how can i do this?  New pull request?